<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterReminder</class>
 <widget class="QMainWindow" name="WaterReminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>298</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>WaterReminder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/trayIcon.png</normaloff>:/Resources/trayIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="labelGlassCount">
        <property name="text">
         <string>You need to drink 0 glasses of water</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDrankGlasses">
        <property name="text">
         <string>Drank 0 Glasses</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelStarted">
      <property name="text">
       <string>Started at 0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Amount to Drink</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="toDrink">
          <property name="suffix">
           <string> mL</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>2500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Hours Awake</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="awakeHours">
          <property name="suffix">
           <string> Hours</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Glass Volume </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="glassVolume">
          <property name="suffix">
           <string> mL</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>9999999</number>
          </property>
          <property name="value">
           <number>250</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="bCalculate">
        <property name="text">
         <string>Calculate and Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bDrankGlass">
        <property name="text">
         <string>Drank Glass</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="labelNextGlass">
        <property name="text">
         <string>Next Glass in 0 Minutes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInterval">
        <property name="text">
         <string>You need to drink a glass of water evey 0 hours</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
